src/GraphODataTemplateWriter/CodeHelpers/CSharp/TypeHelperCSharp.cs

```diff
@@ -319,7 +329,8 @@ public static bool IsComplex(this string t)
         private static readonly Inflector inflector = new Inflector(CultureInfo.GetCultureInfo("en-US"));
         public static string GetNamespaceName(this OdcmNamespace namespaceObject)
         {
-            return inflector.Titleize(namespaceObject.Name).Replace(" ", "");
+            var nameSpaceName = GetReservedNameSpaces().Contains(namespaceObject.Name) ? $"{namespaceObject.Name}NameSpace" : namespaceObject.Name;
```

Comment:
Given that namespace is a single word, can we use Namespace instead of NameSpace?

Reply:
Thanks. Resolved via https://github.com/microsoftgraph/MSGraph-SDK-Code-Generator/pull/668/commits/9c622b62caf0b3c83e46c884d4738646a40dbeb0